app/helpers/course/material/folders_helper.rb

```diff
@@ -0,0 +1,5 @@
+module Course::Material::FoldersHelper
+  def display_name(folder, settings)
+    folder.root? ? settings.title || t('course.material.sidebar_title') : folder.name
```

Comment:
settings is accessible from controller.settings right? Or declare settings to be `helper_method`